test/FunctionalTests/CoreCLRTests/SimpleAppTest_CoreCLR.cs

```diff
@@ -22,7 +22,7 @@ public class SimpleAppTest_CoreCLR :
 
         public ApplicationTestFixture Fixture { get; }
 
-        [Fact(Skip = "Unblocking the build - https://github.com/aspnet/MvcPrecompilation/issues/224")]
+        [Fact]
```

Comment:
Keeping a couple of simple tests, removing all other ones. Most of the flakiness is in deploying and launching the site (port conflicts and timeouts). Fewer tests should help